<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustWindow</class>
 <widget class="QMainWindow" name="CustWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 100, 139);</string>
  </property>
  <widget class="QWidget" name="customer_widget">
   <widget class="QPushButton" name="B_deposit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>    Deposit Funds</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/proje hazırlık/Project-main/deposit.png</normaloff>../../Desktop/proje hazırlık/Project-main/deposit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_withdraw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>    Withdraw Cash</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/proje hazırlık/Project-main/withdraw.png</normaloff>../../Desktop/proje hazırlık/Project-main/withdraw.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_statement">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>Account Statement</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_exit_cust_menu">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_select_opr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 192, 54);
font: 75 22pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>  PLEASE SELECT AN OPERATION</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="la_current_balance">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Current Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_balance">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="B_internal">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>Send Money Internal</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_external">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>Send Money External</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>logo.png</normaloff>logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
